### PR TITLE
Equips medical doctors with medical belts on spawn.

### PR DESCRIPTION
**Pull request by Carbadox**
## About the PR 
Give Medical Doctors medical belts already equipped onto them.

## Why's this needed? 
Medical doctors having to fight over storage space is dumb. This should make things easier for them

## Changelog

```
(u)Carbadox
(*)Medical doctors now start with medical belts.
```

### CODE REVIEW

File: code/datums/jobs.dm [hunk 1]
```diff
@@ -599,15 +599,16 @@
...
 	slot_jump = /obj/item/clothing/under/rank/medical
 	slot_suit = /obj/item/clothing/suit/labcoat
 	slot_foot = /obj/item/clothing/shoes/red
 	slot_lhan = /obj/item/storage/firstaid/regular/doctor_spawn
 	slot_ears = /obj/item/device/radio/headset/medical
 	slot_eyes = /obj/item/clothing/glasses/healthgoggles
-	slot_poc1 = /obj/item/paper/book/pocketguide/medical
-	items_in_backpack = list(/obj/item/crowbar, /obj/item/robodefibrillator) // cogwerks: giving medics a guaranteed air tank, stealing it from roboticists (those fucks)
+	slot_poc1 = /obj/item/device/pda2/medical
+	slot_poc2 = /obj/item/storage/medical_pouch
```
Comment (mordent-goonstation) [inline code/datums/jobs.dm:610]: Hard disagree on adding the pouch. These are intended for Nuke Ops "Combat Medics", not regular doctors.

File: code/obj/item/storage/ammo_pouches.dm [hunk 2]
```diff
@@ -133,6 +133,13 @@
 	/obj/item/item_box/medical_patches/mini_silver_sulf,
 	/obj/item/item_box/medical_patches/mini_synthflesh)
 
+/obj/item/storage/medical_pouch/medic //used by regular medical doctors
+	name = "medical pouch"
+	icon_state = "ammopouch-medic"
+	w_class = 1.00
+	slots = 4
+	does_not_open_in_pocket = 0
```
Comment (mordent-goonstation) [inline code/obj/item/storage/ammo_pouches.dm:141]: Even the nerfed version seems ridiculously good - you're essentially giving doctors another 3 pockets. Strong disagree on adding this.

**Review by mordent-goonstation — CHANGES_REQUESTED** [review]

**Comment by Gerhazo** [comment]
Don't personally think inventory management needs to be trivialized further or that the change's necessary.

Personal opinions aside, as some input in terms of code, with the original currently existing /obj/item/storage/medical_pouch being present for nukeops and an /obj/item/storage/medical_pouch/medic being intended for regular use, it'd probably make more sense to have the architecture be structured the other way around, with /obj/item/storage/medical_pouch for the more general regular pouch and /obj/item/storage/medical_pouch/syndicate (or some other wording of the sort to reflect nuclear operatives, probably in line with what other nukeop-labeled things in code are) for the more specific sort of medical pouch.
At least that's what I think would be more optimal and self-descriptive pathing requiring no use of "used by regular doctors" code comments.

**Comment by Awfullyfishy** [comment]
I strongly agree with the medical belt part, as Engineers and Mechanics start with Utility Belts and yet Medical Staff have to fight over Medical Belts and there's rarely more than 2-3 depending on the map. The pouch I however strongly disagree with. While nice, it's a bit too much.

**Review by mordent-goonstation — APPROVED** [review]

**Pull request merged**